01_Intro/Guess the Number/Guess the Number/Program.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Guess_the_Number
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Wellcome to Guess the Number");
```

Comment:
Ich würde hier eine Funktion draus machen die in etwa "GetDifficulty" heißt. Somit erreichst du besser lesbaren code.